recipes/openai-whisper/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "openai-whisper" %}
+{% set version = "20231117" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/openai-whisper-{{ version }}.tar.gz
+  sha256: 7af424181436f1800cc0b7d75cf40ede34e9ddf1ba4983a910832fcf4aade4a4
+
+build:
+  entry_points:
+    - whisper=whisper.transcribe:cli
+  script: {{ PYTHON }} -m pip install . -vv --no-deps
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - numba
+    - numpy
+    - pytorch
+    - tqdm
+    - more-itertools
+    - tiktoken
+    - ffmpeg
+  run_constrained:
+    - triton >=2.0.0,<3
```

Comment:
I don't know what the best way to add this. Currently, it looks like CUDA is the [only support](https://github.com/conda-forge/triton-feedstock/blob/main/recipe/meta.yaml). 

 [openai/whisper@main/requirements.txt#L7](https://github.com/openai/whisper/blob/main/requirements.txt#L7)

Reply:
Can you add it to the normal run and then we see what the logs say?